packages/plugin-pnp/sources/AbstractPnpInstaller.ts

```diff
@@ -86,7 +78,7 @@ export abstract class AbstractPnpInstaller implements Installer {
       buildScripts.length = 0;
     }
 
-    const packageFs = !hasVirtualInstances && pkg.linkType !== LinkType.SOFT
+    const packageFs = pkg.linkType !== LinkType.SOFT
```

Comment:
This change means that all packages with peer dependencies will be unplugged once more (so even if there is a single instance, there will be two unplugs).

Reply:
If this is true, then this is a bug that has existed for a long time. 

`hasVirtualInstances` is always `false` so it was dead code.